share/availability/light/availability_test.go

```diff
@@ -285,6 +286,7 @@ func TestSharesRoundTrip(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			// prepare data
 			b.Data.Messages.MessagesList = make([]core.Message, len(tc.messages))
+			b.OriginalSquareSize = 16
```

Comment:
I've never seen this param before @evan-forbes can you explain how it works? Also, tests work if we decrease OG square size to 8.

Reply:
This square size is effectively a cached value that is normally filled out by block producers, so unless we want to have an instance of the application producer the block for us, we have to fill it out by hand.

Reply:
come to think of it, a lot of the logic is stateless, so we should be able to abstract it out for anyone to call... :thinking:

Reply:
Okay I see why OriginalSquareSize also works with 8 -- bc with 16, it had more tail share padding.

Reply:
feel free to resolve @evan-forbes